packages/core/src/reporters/mutation-test-report-helper.ts

```diff
@@ -97,6 +97,7 @@ export class MutationTestReportHelper {
   }
 
   private reportOne(result: MutantResult): MutantResult {
+    result.location = this.toLocation(result.location);
```

Comment:
I understand that this is more DRY than doing it in `reportMutantRunResult`, but the result is that the construction of `result` is now spread across 2 places. Also you're mutating an object on the heap, which can lead to undesired consequences (it's convoluted).

Can you move this to `reportMutantRunResult`? You can still do the location calculation once atop that function.

Reply:
Yes, that's probably better than the 2-staged way of doing now, do you agree?

Reply:
Seems good, since we keep the creation/modification outside of the `reportOne` method (that should only report and not do anything else).